### ISSUE TITLE
Invalid Cannot decrypt / verify JWT error message

### ISSUE THREAD

**Issue by pditommaso** [micronaut-projects/micronaut-security#75]
When using JWT along with basic authentication method and the latter is used, the error following error message is reported in the log file: 

```
Cannot decrypt / verify JWT: Invalid JWT serialization: Missing dot delimiter
```

My understanding is that MN tries to decode the Basic auth token as it were a JWT token, tehn fails and fallback on the basic auth provider.

**Comment by jameskleeh** [comment]
Fixed by https://github.com/micronaut-projects/micronaut-security/commit/e2c90d0533ccac922656ce8fb032321ba69c4389

**Comment by pditommaso** [comment]
Has this been included in the release 1.2.1 by any chance?

**Comment by jameskleeh** [comment]
Yes
